From c1f38d6b8acfdac4a78a30372d3be5708e5970ad Mon Sep 17 00:00:00 2001
From: Jon TURNEY <jon.turney@dronecode.org.uk>
Date: Sun, 22 Nov 2009 23:35:09 +0000
Subject: [PATCH 1/2] Cygwin/X: Avoid cursor size log spam

Fedora 12 likes to use a 39x26 animated wait cursor.  Avoid spamming
the log with warnings that each frame can't be completely contained
in the 32x32 native cursor

Also reformat log message so it doesn't contain a '\n\t'. I mean,
it's not like we might want to grep the log or something...

Signed-off-by: Jon TURNEY <jon.turney@dronecode.org.uk>
---
 hw/xwin/wincursor.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/hw/xwin/wincursor.c b/hw/xwin/wincursor.c
index ce98162..7f1935a 100644
--- a/hw/xwin/wincursor.c
+++ b/hw/xwin/wincursor.c
@@ -188,8 +188,7 @@ winLoadCursor (ScreenPtr pScreen, CursorPtr pCursor, int screen)
   if (pScreenPriv->cursor.sm_cx < pCursor->bits->width || 
       pScreenPriv->cursor.sm_cy < pCursor->bits->height)
     {
-      winErrorFVerb (2, "winLoadCursor - Windows requires %dx%d cursor\n"
-	      "\tbut X requires %dx%d\n",
+      winErrorFVerb (3, "winLoadCursor - Windows requires %dx%d cursor but X requires %dx%d\n",
 	      pScreenPriv->cursor.sm_cx, pScreenPriv->cursor.sm_cy,
 	      pCursor->bits->width, pCursor->bits->height);
     }
-- 
1.6.4.2

